listenbrainz_spark/RabbitMQ/rabbitmq_connection.py

```diff
@@ -0,0 +1,61 @@
+import pika
+import logging
+from listenbrainz_spark import config
+import time
+import json
+import sys
+
+class SparkReader():
```

Comment:
A class called SparkReader() in the rabbitmq_connection is weird to me. It is specifically used to write queried user stats to RabbitMQ... Should it be called StatsWriter() or RabbitMQWriter()? And should the module get a better name?

Reply:
StatsWriter and the module `stats_writer/stats_writer.py` would look good to me.

Reply:
Agreed!

Reply:
I will say StatsWriter. What about you? 
Would you like to suggest any name for the module?

Reply:
As param suggested, would be great!